doc/doc-requirements.txt

```diff
@@ -1,6 +1,6 @@
 # documentation specific extra packages
 sphinx>=1.7
```

Comment:
If i recall correctly, having a dedicated entry for sphinx as well as the order of entriers matters greatly in this file to make everything work smoothly on RTD. This relates to RTD coming with a pre-installed sphinx version and given how `pip` will try to please the various requirements.

This is from z2jh's `requirements.txt, which followed https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1658 and https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/1723.

```shell
# Order matters:
# - myst-parser depends on "sphinx>=2,<3"
# - pydata-sphinx-theme depends on "sphinx"
# - sphinx-copybutton depends on "sphinx>=1.8"
#
# Listing either pydata-sphinx-theme or sphinx-copybutton first will make the
# myst-parser constraints on sphinx be ignored, so myst-parser should go
# first. This is only relevant if sphinx==1.* is already installed in the
# environment, which it is on ReadTheDocs.
#
myst-parser
pydata-sphinx-theme
pyyaml
sphinx-autobuild
sphinx-copybutton
```

```suggestion
```